src/http/fetcher.model.ts

```diff
@@ -127,7 +127,7 @@ export interface FetcherRequest
   timeoutSeconds: number
   retry: FetcherRetryOptions
   retryPost: boolean
-  retry3xx: boolean
+  retry3xx?: boolean
```

Comment:
Without better understanding of the repo, this feels hacky.

TS was complaining that `retry3xx` is optional in `FetcherOptions` and mandatory in `FetcherRequest`
This `?` solves the complaint, but probably all the other fields should match. We should probably fix the rest, but I am afraid of a rabbit's hole.

```ts
interface FetcherRequest
  extends Omit<FetcherOptions, 'method' | 'headers' | 'baseUrl' | 'url'>  {
  retry3xx: boolean
  // ...

export interface FetcherOptions {
  retry3xx?: boolean
  // ...
}
}
```